src/apps/search-result/search-result.entry.tsx

```diff
@@ -1,40 +1,34 @@
 import * as React from "react";
 import { getUrlQueryParam } from "../../core/utils/helpers";
-import { getNumberOfResultItems } from "./helpers";
+import { getPageSize } from "./helpers";
 import SearchResult from "./search-result";
 
-export interface ResultItemsProps {
-  numberOfResultItemsDesktop?: number;
-  numberOfResultItemsMobile?: number;
-}
-
-export interface SearchResultEntryProps extends ResultItemsProps {
+export interface SearchResultEntryProps {
   q?: string;
+  pageSizeDesktop?: number;
+  pageSizeMobile?: number;
 }
 
 const SearchResultEntry: React.FC<SearchResultEntryProps> = ({
   q: attrQ,
-  numberOfResultItemsDesktop: attrNumberOfResultItemsDesktop,
-  numberOfResultItemsMobile: attrNumberOfResultItemsMobile
+  pageSizeDesktop: attrPageSizeDesktop,
+  pageSizeMobile: attrPageSizeMobile
 }) => {
   // If a q string has been defined as a data attribute use that
   // otherwise use the one from the url query parameter.
   const searchQuery = attrQ || (getUrlQueryParam() as string);
   // Get number of result items to be shown.
   // If the number of items has been defined with data attributes use those
   // otherwise get them from the configuration.
-  const numberOfResultItems = getNumberOfResultItems({
-    desktop: attrNumberOfResultItemsDesktop,
-    mobile: attrNumberOfResultItemsMobile
+  const pageSize = getPageSize({
+    desktop: attrPageSizeDesktop,
+    mobile: attrPageSizeMobile
   });
 
   return (
     <div>
       {searchQuery && (
-        <SearchResult
-          q={searchQuery}
-          numberOfResultItems={numberOfResultItems as number}
-        />
+        <SearchResult q={searchQuery} pageSize={pageSize as number} />
       )}
     </div>
```

Comment:
If I change it to a fragment I get the eslint error:
"Fragments should contain more than one child - otherwise, there‘s no need for a Fragment at all."
I'll keep it for now...